### ISSUE TITLE
API responded with status 401!

### ISSUE THREAD

**Issue by danishleh** [victornpb/undiscord#559]
### Undiscord Version

v5.2.1

### Browser

Firefox 111.0.1 (64-bit)

### Extension

TamperMonkey 4.18.1 2022-11-13

### Reproduction steps

1. Open browser and navigate to discord.com.
2. Open UnDiscord.
3. Open a DM you want to wipe/delete.
4. Filled required boxes to perform wipe. (Author ID, Server ID, Channel ID).
5. Attempt to delete messages.

### Actual results

Started at 4/2/2023, 6:18:31 PM
authorId = "[REDACTED]" guildId = "[REDACTED]" channelId = "[REDACTED]" minId = "[REDACTED]" maxId = "[REDACTED]" hasLink = false hasFile = false
Fetching messages...
Error searching messages, API responded with status 401!
{"message":"401: Unauthorized","code":0}
CoreException {}

### Expected results

For it to properly delete my messages.

### Bug affects

- [X] DMs
- [ ] DMs (group)
- [ ] Channels
- [ ] Channel (NSFW)

### Additional information & file uploads

I did manually fill my token after initially encountering this issue, but it did not have any effect on the actual results. I'm at a loss for what's causing this error.*

**Comment by larryqiann** [comment]
You might have to manually clear out all of the fields if your browser has autofill enabled. Then, hit "fill token" again.
The problem appears to be that some of the fields, including the hidden by default "message range" ones, are detected as passwords and the browser autofills garbage into them. I'm not sure if this is the problem in your case but this fixed it for me.

**Comment by Noreliks** [comment]
Столкнулся с той же проблемой, но нашёл решение. Это всё из-за автозаполнения полей браузером. Чтобы решить проблему, нужно: 1. В окне скрипта, есть вкладка "General", тут нужно нажать me, current, current, чтобы они заполнились. 2. Вкладка "Messages Interval", тут надо стереть всё что браузер сам вписал и оставить поля пустыми. 3. Вкладка "Advanced Settings" тут стираем всё что вписано в поле Authorization token и нажимаем fill. 4. Готово. Можно снова запускать скрипт.

**Comment by vreoo** [comment]
> You might have to manually clear out all of the fields if your browser has autofill enabled. Then, hit "fill token" again. The problem appears to be that some of the fields, including the hidden by default "message range" ones, are detected as passwords and the browser auto-fills garbage into them. I'm not sure if this is the problem in your case but this fixed it for me.

@larryqiann That's true, I have faced the same issue, and I found out that some fields were auto-filled. So I disabled auto-fill and cleared all the fields and it worked just fine for me.

**Comment by false-fox** [comment]
Autofilled fix in #631
